lib/NormalModuleFactory.js

```diff
@@ -1017,12 +1013,14 @@ If changing the source code is not an option there is also a resolve options cal
 						}
 						if (err) return callback(err);
 
-						const parsedResult = identToLoaderRequest(result);
+						const parsedResult = this._parseResourceWithoutFragment(result);
```

Comment:
we can pass `cachedParseResourceWithoutFragment` as an argument, but I think this would be a breaking change